tensorboard/uploader/uploader.py

```diff
@@ -140,6 +154,50 @@ def _upload_once(self):
         self._request_sender.send_requests(run_to_events)
 
 
+def update_experiment_metadata(
+    writer_client, experiment_id, name=None, description=None
+):
+    """Modifies user data associated with an experiment.
+
+    Args:
+      writer_client: a TensorBoardWriterService stub instance
+      experiment_id: string ID of the experiment to modify
+      name: If provided, modifies name of experiment to this value.
+      description: If provided, modifies the description of the experiment to
+         this value
+
+    Raises:
+      ExperimentNotFoundError: If no such experiment exists.
+      PermissionDeniedError: If the user is not authorized to modify this
+        experiment.
+      RuntimeError: On unexpected failure.
```

Comment:
Is this superfluous? The method doesn’t appear to raise `RuntimeError`
directly, and “we might raise an arbitrary exception due to a
programming error” isn’t really part of the public API (cf. [style guide
on “Raises”][sty]).

[sty]: https://google.github.io/styleguide/pyguide.html#doc-function-raises

Reply:
Done.  This was cloned to be consistent with  `delete_experiment` below

Reply:
Updated.  This was here by copying from above.